CHANGES.rst

```diff
@@ -14,7 +14,8 @@ Changes to API
 Bug Fixes
 ---------
 
-- 
+- Implement byteorder swap method that is forward-compatible with
+  numpy 2.0 in jwst ramp_fitting. [#282]
```

Comment:
```suggestion
```
#284 switched change log handling to `towncrier`, so this change log entry should be a file in `changes/`:
```shell
echo "Implement byteorder swap method that is forward-compatible with numpy 2.0 in jwst ramp_fitting." > changes/282.bugfix.rst
```

(new PRs will have these instructions in the checklist)